lib/mintel/dashboards/components/panels/container_resources.libsonnet

```diff
@@ -0,0 +1,38 @@
+local layout = import 'components/layout.libsonnet';
+local commonPanels = import 'components/panels/common.libsonnet';
+local promQuery = import 'components/prom_query.libsonnet';
+{
+ 
+  containerResourcesPanel(podSelectorValue, containerName='main', startRow=1000)::
```

Comment:
feels like we are already duplicating this very same row , 

* https://github.com/mintel/kube-monitoring-mixins/blob/master/lib/mintel/dashboards/components/panels/thumbor.libsonnet#L19
* https://github.com/mintel/kube-monitoring-mixins/blob/master/lib/mintel/dashboards/components/panels/django.libsonnet#L33

Reply:
Yeah, this is related to my description. 

I noted this too, so I introduced container-resources as a shared component. 

I want to update django/thumbor , but in a different PR (because this one is all omni-related)

Reply:
Updated to share containerResourcesPanel